packages/lesmis/src/views/MapView.js

```diff
@@ -17,27 +33,137 @@ const Container = styled.div`
   min-height: 600px;
 `;
 
+const Map = styled(MapContainer)`
+  flex: 1;
+  height: 100%;
+`;
+
 const Main = styled.div`
   position: relative;
   flex: 1;
   height: 100%;
 `;
 
+const EmptyPanel = styled(MapOverlaysPanelContainer)`
+  padding: 2rem;
+`;
+
+const Legend = styled(LegendComponent)`
+  position: absolute;
+  bottom: 5px;
+  z-index: 999;
+  left: 50%;
+  transform: translateX(-50%);
+`;
+
+const TilePicker = styled(TilePickerComponent)`
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  z-index: 9999;
+  pointer-events: none;
+`;
+
+const useProvinceData = () => {
+  const query = useEntitiesData();
+  let provinces = [];
+
+  if (query.data) {
+    provinces = query.data.filter(entity => entity.parentCode === 'LA');
```

Comment:
```suggestion
    provinces = query.data.filter(entity => entity.type === 'district');
```
Weirdly, there are some facilities that have Laos as their parent

![Screen Shot 2021-04-16 at 2 10 40 PM](https://user-images.githubusercontent.com/1274422/114961807-a473eb80-9ebd-11eb-854f-e0588a174070.png)

Reply:
I guess it's a bit strange to use `district` as the comparison value when we're looking for provinces, so maybe a variable called `BACKEND_PROVINCE_TYPE` or something to make sure people don't think it's a bug/typo